config/api.yml

```diff
@@ -1789,6 +1792,7 @@
     - :vms
     - :flavors
     - :cloud_templates
+    - :container_images
```

Comment:
We need some tests exercising the behavior here - what can/can't you do with container images of providers?

Reply:
I am sorry, this should have been moved.